discovery-provider/integration_tests/premium_content/test_access.py

```diff
@@ -23,7 +23,11 @@ def test_access(app):
                 "nft_collection": {
                     "chain": "eth",
                     "standard": "ERC721",
-                    "address": "some-nft-collection",
+                    "address": "some-nft-collection-address",
+                    "name": "some nft collection name",
+                    "slug": "some-nft-collection",
+                    "externalLink": "some-nft-collection-external-link",
+                    "permaLink": "some-nft-collection-perma-link",
```

Comment:
nit: `permalink` is what we use elsewhere

Reply:
removed it here as we dont need it anymore